jsvg-javafx/src/main/java/com/github/weisj/jsvg/ui/jfx/FXSVGCanvas.java

```diff
@@ -0,0 +1,301 @@
+        timeline.setCycleCount(Timeline.INDEFINITE);
+        timeline.playFromStart();
+    }
+
+    ////////////////////////////////////////////////
+
+    /**
+     * Requests that the SVG Document be repainted
+     * In most cases this is not necessary, as the canvas will automatically be repainted when any properties (e.g. document, view-box, backend) are changed
+     */
+    public void repaint() {
+        SkinBase<?> skin = (SkinBase<?>) getSkin();
+        if (skin != null) {
+            FXSVGCanvasSkin fxSkin = (FXSVGCanvasSkin) skin;
+            fxSkin.markDirty();
+        }
+    }
+
+    public void playAnimation() {
+        timeline.play();
+    }
+
+    public void pauseAnimation() {
+        timeline.pause();
+    }
+
+    public void restartAnimation() {
+        timeline.playFromStart();
+    }
+
+    public void stopAnimation() {
+        timeline.stop();
+    }
+
+    ////////////////////////////////////////////////
+
+    private final ObjectProperty<@NotNull RenderBackend> renderBackend =
+            new SimpleObjectProperty<>(DEFAULT_RENDER_BACKEND);
+
+    public @NotNull RenderBackend getRenderBackend() {
+        return renderBackend.get();
+    }
+
+    public @NotNull ObjectProperty<@NotNull RenderBackend> renderBackendProperty() {
+        return renderBackend;
+    }
+
+    public void setRenderBackend(@NotNull RenderBackend renderer) {
+        this.renderBackend.set(renderer);
+    }
+
+    ////////////////////////////////////////////////
+
+    private final ObjectProperty<@Nullable SVGDocument> document = new SimpleObjectProperty<>();
+
+    public @Nullable SVGDocument getDocument() {
+        return document.get();
+    }
+
+    public @NotNull ObjectProperty<@Nullable SVGDocument> documentProperty() {
+        return document;
+    }
+
+    public void setDocument(@Nullable SVGDocument document) {
+        this.document.set(document);
+    }
+
+    ////////////////////////////////////////////////
+
+    private final BooleanProperty useSVGViewBox = new SimpleBooleanProperty(DEFAULT_USE_SVG_VIEW_BOX);
+
+    public boolean isUsingSVGViewBox() {
+        return useSVGViewBox.get();
+    }
+
+    public BooleanProperty useSVGViewBoxProperty() {
+        return useSVGViewBox;
+    }
+
+    public void setUseSVGViewBox(boolean useSVGViewBox) {
+        this.useSVGViewBox.set(useSVGViewBox);
+    }
+
+    ////////////////////////////////////////////////
+
+    private final ObjectProperty<@Nullable ViewBox> currentViewBox = new SimpleObjectProperty<>();
+
+    public @Nullable ViewBox getCurrentViewBox() {
+        return currentViewBox.get();
+    }
+
+    public @Nullable ReadOnlyObjectProperty<@Nullable ViewBox> currentViewBoxProperty() {
+        return currentViewBox;
+    }
+
+    ////////////////////////////////////////////////
+
+    private final ObjectProperty<@Nullable ViewBox> viewBox = new SimpleObjectProperty<>(DEFAULT_VIEW_BOX);
+
+    public @Nullable ViewBox getViewBox() {
```

Comment:
`viewBox()`

Reply:
In general I like to name accessors without the `get` (though if JavaFX uses the other convention feel free to ignore these comments)

Reply:
Yes same as above, JavaFX style conventions generally keep the get